starlite/testing.py

```diff
@@ -4,7 +4,13 @@
 from orjson import dumps
 from pydantic import BaseModel
 from pydantic.typing import AnyCallable
-from requests.models import RequestEncodingMixin
+
+try:
+    from requests.models import RequestEncodingMixin
+except ImportError:
+    raise RuntimeError(
```

Comment:
We have an exception for this case `MissingDependencyException`, see for example `starlite/plugins/sql_alchemy.py`